src/components/ReservatioPage.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import SingleReservation from './SingleReservation';
+import Logo from './Logo.js';
+import Toolbar from './Toolbar';
+
+export default class ReservatioPage extends Component {
+    constructor(props) {
+        super(props)
+    
+        this.state = {
+             haircutDetails:[{ id:1,timezone:"11:00 - 11:20", status:"free"},
+             {id:2,timezone:"11:20 - 11:40", status:"free"},
+             {id:3,timezone:"11:40 - 12:00", status:"taken"},
+             {id:4,timezone:"12:00 - 12:20", status:"free"},
+             {id:5,timezone:"12:20 - 12:40", status:"cancel"},
+             {id:6,timezone:"12:40 - 13:00", status:"free"},
+             {id:7,timezone:"13:00 - 13:20", status:"free"},
+             ],
+             Days:["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
```

Comment:
I didnt really meant this. It was supposed to be state.Day: "Sunday"
So next step would be getting it from the props
but for now its fine